src/ray/raylet/local_object_manager.cc

```diff
@@ -340,16 +377,19 @@ void LocalObjectManager::AddSpilledUrls(
     request.set_object_id(object_id.Binary());
     request.set_spilled_url(object_url);
     request.set_spilled_node_id(node_id_object_spilled.Binary());
-    request.set_size(it->second.first->GetSize());
-
-    auto owner_client = owner_client_pool_.GetOrConnect(it->second.second);
-    RAY_LOG(DEBUG) << "Sending spilled URL " << object_url << " for object " << object_id
-                   << " to owner " << WorkerID::FromBinary(it->second.second.worker_id());
-    // Send spilled URL, spilled node ID, and object size to owner.
-    owner_client->AddSpilledUrl(
-        request, [unpin_callback](Status status, const rpc::AddSpilledUrlReply &reply) {
-          unpin_callback(status);
-        });
+    request.set_size(it->second->GetSize());
+    const auto &owners = util::get_ref_or_fail(object_owners_, object_id);
+    for (auto &owner : owners) {
```

Comment:
```suggestion
    for (const auto &owner : owners) {
```

Reply:
i think here  `auto` == `const auto` since owners is const.